src/main/java/org/folio/circulation/infrastructure/storage/inventory/ItemRepository.java

```diff
@@ -70,6 +71,7 @@
   private final boolean fetchLoanType;
 
   private static final String ITEMS_COLLECTION_PROPERTY_NAME = "items";
+  private static final String HOLDINGS_RECORDS_PREPERTY_NAME = "holdingsRecords";
```

Comment:
just correct like PROPERTY (it is PREPERTY)

Reply:
done